### PR TITLE
A working dissector for protobuf

### PR DESCRIPTION
**Pull request by Elyash**

### CODE REVIEW

File: modules/generic/proto.lua [hunk 1]
```diff
@@ -32,7 +32,7 @@ local GenericExperts     = require "generic.experts"
 local GenericProto = {}
 
 
-GenericProto.proto = Proto.new("Protobuf", "Google Protobuf Format")
+GenericProto.proto = Proto.new("Protobuf3", "Google Protobuf Format")
```
Comment (yelly) [inline modules/generic/proto.lua:35]: Isn't this supposed to still be for proto2?

File: modules/generic/proto_fields.lua [hunk 2]
```diff
@@ -33,15 +33,15 @@ local proto_pfields = {
     tag      = ProtoField.uint32("protobuf.tag", "Tag", base.DEC, nil, nil, "Field tag value"),
     wiretype = ProtoField.uint8 ("protobuf.wiretype", "Wire Type", base.DEC, wiretype_valstring, 0x07, "Field wire type value"),
     length   = ProtoField.uint32("protobuf.length", "Length", base.DEC, nil, nil, "Field length"),
-    value    = ProtoField.bytes ("protobuf.value", "Value", base.HEX, nil, nil, "Field value bytes"),
+    value    = ProtoField.bytes ("protobuf.value", "Value", base.NONE, nil, nil, "Field value bytes"),
```
Comment (yelly) [inline modules/generic/proto_fields.lua:36]: Why? Do you expect this field to always be NULL?

File: modules/generic/proto_fields.lua [hunk 3]
```diff
@@ -33,15 +33,15 @@ local proto_pfields = {
...
     length_delimiter = ProtoField.uint32("protobuf.length_delimiter_value", "Length Delimiter Value", base.DEC, nil, nil, "Field length-delimited length value"),
 
     -- following are for wiretypes for unknown fields
     FIXED32  = ProtoField.uint32("protobuf.fixed32", "Fixed32 Value", base.DEC, nil, nil, "Unknown Fixed32 Field value"),
     FIXED64  = ProtoField.uint64("protobuf.fixed64", "Fixed64 Value", base.DEC, nil, nil, "Unknown Fixed64 Field value"),
     VARINT   = ProtoField.uint64("protobuf.varint", "Varint Value", base.DEC, nil, nil, "Unknown Varint Field value"),
     GROUP    = ProtoField.bytes ("protobuf.group", "Group", base.NONE, nil, nil, "Unknown Group Field"),
-    LENGTH_BYTES = ProtoField.bytes("protobuf.length_delimited_bytes", "Length Delimited Bytes", base.HEX, nil, nil, "Unknown Length Delimited Field"),
+    LENGTH_BYTES = ProtoField.bytes("protobuf.length_delimited_bytes", "Length Delimited Bytes", base.NONE, nil, nil, "Unknown Length Delimited Field"),
```
Comment (yelly) [inline modules/generic/proto_fields.lua:44]: Why? Do you expect this field to always be NULL?

**Review by yelly — CHANGES_REQUESTED** [review]
CR 1.

File: modules/generic/proto.lua (hunk 1 above)
Comment (Elyash) [reply modules/generic/proto.lua:35]: I am using both Repos so I need to distinguish between their name...

File: modules/generic/proto_fields.lua (hunk 2 above)
Comment (Elyash) [reply modules/generic/proto_fields.lua:36]: I will fix this later.
I get an error so bytes object can't get `base.HEX`

File: modules/generic/proto_fields.lua (hunk 3 above)
Comment (Elyash) [reply modules/generic/proto_fields.lua:44]: I will fix this later.
I get an error so bytes object can't get `base.HEX`

**Review by yelly — APPROVED** [review]
OK.

**Pull request merged**